ocfweb/tv/templates/tv/tv-base.html

```diff
@@ -0,0 +1,30 @@
+{% load staticfiles %}
```

Comment:
i'd just call this file `tv/base.html`, it's not ambiguous because of the `tv/` prefix